### ISSUE TITLE
Improve DXF usability

### ISSUE THREAD

**Issue by d1nuc0m** [floookay/adelheid#5]
While working on a 3D printed case, I've imported the lasercut files into LibreCAD, QCAD and FreeCAD and found some minor problems that make it difficult to edit them. These are the possible improvements:

- Remove stacked lines (sometimes there are two lines in the same place, this is a problem when using some online manufacturing services)
- Fillets/rounds and hexagons are represented with polilines, replace them with circles/arcs/hexagons (it is a problem when importing in 3D)
- Create polilines (e.g. a button hole should be a poliline, otherwise importing in 3D CADs leads to "open wires" errors)

(note: for the case I'm already modifying the DXFs, if I achieve to complete the case I'll also upload them)

**Comment by floookay** [comment]
Yeah, that's a problem I also had when I last tried to work with them. These are the files Fusion 360 created which doesn't work on my machine anymore unfortunately. I created the dxf and svg files starting with `z_hipro_layers` in Inkscape which I also used to order the acrylic.

If you can achieve to fix them that would be greatly appreciated! :heart:

**Comment by d1nuc0m** [comment]
> These are the files Fusion 360 created 

In Fusion did you create a 3D model and extracted the sketches or did you jus draw in 2D "AutoCAD style"? Also, did you work in inches or mm? (I get some strange measurements in mm, I'd like to understand if it is wanted)

> If you can achieve to fix them that would be greatly appreciated!

I think I have already _partially_ fixed them in LibreCAD/QCAD, but I have to check for other issues (and document how to import them in FreeCAD). As soon as they're ready, I'll open a PR

**Comment by floookay** [comment]
I modified the Fusion files of the Arisu. These were drawn in the sketch mode, no 3d. I'm also fairly certain I was doing the adjustments in mm as well.
